src/components/NetworkModal/index.tsx

```diff
@@ -223,28 +225,28 @@ export default function NetworkModal(): JSX.Element | null {
                                     [
                                         ChainId.MAINNET,
                                         ChainId.KOVAN,
-                                        ChainId.RINKEBY,
+                                        ChainId.RINKEBY, 
                                         ChainId.GÖRLI,
                                         ChainId.ROPSTEN
                                     ].includes(key as any)
                                 ) {
                                     console.log(key.toString(16))
-                                    if (isMetaMask) {
-                                        ; (ethereum as any).request({
-                                            method: 'wallet_switchEthereumChain',
-                                            params: [{ chainId: `0x${key.toString(16)}` }]
-                                        })
+                                    if (cLibrary.isMetaMask) {
```

Comment:
what if it is coinbase?

Reply:
coinbase doesn't work with the request method but with the .send
metamask would work with both, but their docs state it as 'deprecated'